docs/docs/standard-lib/env.md

```diff
@@ -24,7 +24,7 @@ To make use of the Env module an import is required.
 import Env;
 ```
 
-### Env.get(string, string: defaultValue -> optional)
+### Env.get(String, String: defaultValue -> Optional) -> String -> Nil
```

Comment:
```suggestion
### Env.get(String, String: defaultValue -> Optional) -> ?String
```

I think this could be a nice way to denote a nil-able value, what do you think?